src/main/java/org/folio/consortia/service/TenantService.java

```diff
@@ -58,4 +59,13 @@ public interface TenantService {
    * @return central tenant id
    */
   String getCentralTenantId();
+
+  /**
+   * Creates user_tenant.
+   *
+   * @param userTenant user_tenant.
+   *
+   * @return userTenant
+   */
+  UserTenant postUserTenant(UserTenant userTenant);
```

Comment:
Lets not expose this method to contract, just invoke userTenantClient from TenantService,
because this method will not be used by any other places, just when settupping tenant

Reply:
fixed